kong-2.8.0-0.rockspec

```diff
@@ -65,15 +65,13 @@ build = {
     ["kong.conf_loader.listeners"] = "kong/conf_loader/listeners.lua",
 
     ["kong.clustering"] = "kong/clustering/init.lua",
-    ["kong.clustering.version_negotiation"] = "kong/clustering/version_negotiation/init.lua",
-    ["kong.clustering.version_negotiation.services_known"] = "kong/clustering/version_negotiation/services_known.lua",
-    ["kong.clustering.version_negotiation.services_requested"] = "kong/clustering/version_negotiation/services_requested.lua",
```

Comment:
when is this coming back?

Reply:
Without these, this PR is doing more than a refactor.

Reply:
After this PR, we will try another method to implement similar feature.